.sandbox/manifest.yaml

```diff
@@ -0,0 +1,4 @@
+daemons:
+  app:
+    run:
+      cmd: 'cd spring-boot && ./mvnw spring-boot:run'
```

Comment:
Please avoid using `cd`. There's a property called `dir`:

```yaml
daemons:
  app:
     run:
        cmd: ...
        dir: spring-boot
```

Reply:
I see, perfect! I'll update it shortly.